docs/_articles/en/codebuilder/cb-diffs.md

```diff
@@ -2,23 +2,19 @@
 title: What's Different
 lang: en
 ---
+
 ## What’s Different in Code Builder
 
 Code Builder unleashes the power of Salesforce Extensions for VS Code and offers the same rich features as the desktop version. No matter whether you're using VS Code on your desktop or Code Builder from a browser, the Salesforce extensions you access are the same. While the extensions generally function the same way regardless of how they're accessed, there are some nuances to be aware of:
 
-* Code Builder is installed as a managed package, and it comes with everything you need: VS Code, Salesforce extensions, and Salesforce CLI. No installation required. 
-* Unlike VS Code for the desktop, you can't add any new extensions to your Code Builder development environment.
```

Comment:
@sbudhirajadoc - This one is incorrect.  Correct bullet point for this one (feel free to change the wording, of course): "The extensions available to add to your Code Builder environment include anything published to the Open VSX public forum.  Unlike VS Code for desktop, you cannot access the Microsoft Marketplace of extensions so you may see differences in the extensions available."